source/index.txt

```diff
@@ -22,76 +22,61 @@ Introduction
-to MongoDB and work with your data. On this page, you can find descriptions of
-each section of the driver documentation and where to go to learn more
-about the Go driver.
+Connect your Go applications to MongoDB and work with your data using
+the official Go driver. On this page, you can find descriptions of each
+section of the driver documentation and how to learn more about the Go
+driver.
 
 Quick Start
 -----------
 
-The :doc:`Quick Start </quick-start>` section is where you can
-establish a connection to MongoDB Atlas and begin working with data.
+Learn how to establish a connection to MongoDB Atlas and begin
+working with data in the :doc:`Quick Start </quick-start>` section.
 
 Usage Examples
 --------------
 
-The :doc:`Usage Examples </usage-examples>` section is where you can
-find runnable code snippets and high-level explanations for common
-methods.
+For fully runnable code snippets and explanations for common
+methods, see the :doc:`Usage Examples </usage-examples>` section.
 
 Fundamentals
 ------------
 
-The Fundamentals section explains how to perform the following tasks
-using the Go driver:
-
-- :doc:`Connect to MongoDB </fundamentals/connection>`
-- :doc:`How the Driver Uses Context </fundamentals/context>`
-- :doc:`Authenticate with MongoDB </fundamentals/auth>`
-- :doc:`Read from and Write to MongoDB </fundamentals/crud>`
-- :doc:`Work with BSON </fundamentals/bson>`
-
-.. - :doc:`Specify an API Version </fundamentals/versioned-api>`
-.. - :doc:`Use the Driver's Data Formats </fundamentals/data-formats>`
-.. - :doc:`Perform Aggregations </fundamentals/aggregation>`
-.. - :doc:`Construct Indexes </fundamentals/indexes>`
-.. - :doc:`Specify Collations </fundamentals/collations>`
-.. - :doc:`Record Events in the Driver </fundamentals/logging>`
-.. - :doc:`Use Driver Events in your Code </fundamentals/monitoring>`
-.. - :doc:`Store and Retrieve Files in MongoDB </fundamentals/gridfs>`
+.. include:: /includes/fundamentals-sections.rst
 
 API
 ---
 
-The `MongoDB Go driver API documentation <{+api+}/mongo>`__ contains
-detailed information about types and functions in the MongoDB Go driver.
+For detailed information about types and functions in the MongoDB
+Go driver, see the `MongoDB Go driver API documentation
+<{+api+}/mongo>`__.
 
 .. FAQ
 .. ---
 
-.. The :doc:`Frequently Asked Questions (FAQ) </faq>` section provides answers
-.. to commonly asked questions about the MongoDB Go Driver.
+.. For answers to commonly asked questions about the MongoDB
+.. Go Driver, see the :doc:`Frequently Asked Questions (FAQ) </faq>`
+.. section.
 
 Issues & Help
 -------------
 
-The :doc:`Issues & Help </issues-and-help>` section explains how to report
-bugs, contribute to the driver, and find additional resources for asking
-questions and receiving help.
+Learn how to report bugs, contribute to the driver, and find
+additional resources for asking questions and receiving help in the
```

Comment:
```suggestion
additional resources for asking questions in the
```
Since this is already a list containing an "and" before this list item, and "and receiving help" doesn't add any extra information to the list item, I suggest that we should eliminate it to reduce wordiciousness. Up to you, though -- maybe there's some context behind this wording that I don't have.